server/graphql/dog-licenses.js

```diff
@@ -52,69 +74,120 @@ type LicensesArgs = {
   ids: string[],
 };
 
+//must match graphql schema above
 type DogLicense = {
-  id: string,
   firstName: string,
   lastName: string,
-  deathDate: ?string,
-  deathYear: string,
-  pending: boolean,
-  age: ?string,
-};
-
-type DogLicenseSearch = {
-  page: number,
-  pageSize: number,
-  pageCount: number,
-  results: DogLicense[],
-  resultCount: number,
+  dogName: string,
+  id: string,
+  year: number,
+  personid: string,
+  tagNo: string,
+  tagType: string,
+  tagExp: number,
+  tagStat: string,
+  vacDate: number,
+  vacTerm: number,
+  vacExp: number,
+  userid: string,
+  stamp: number,
+  jurisdiction: string,
+  tagIdentity: number,
+  tagTail: string,
+  clerkid: string,
+  animalType: string,
+  sex: string, //?
+  yearsOld: number,
+  monthsOld: number,
+  dob: number, //?
+  colorGroup: string,
+  priColor: string,
+  secColor: string,
+  breedGroup: string,
+  priBreed: string,
+  secBreed: string,
+  animalCond: string,
+  animalStat: string,
+  statusDate: number,
+  animalSize: string,
+  streetNo: number,
+  streetName: string,
+  streetType: string,
+  apt: string, //?
+  city: string,
+  state: string,
+  zipCode: string,
+  areaCode: string,
+  phone: string,
 };
 
 function searchResultToDogLicense(
   res: DogLicenseSearchResult | DbDogLicense,
 ): DogLicense {
+  //console.log('RES', res);
   return {
-    id: res.CertificateID.toString(),
-    firstName: res['First Name'],
-    lastName: res['Last Name'],
-    deathDate: res['Date of Death'],
-    deathYear: res.RegisteredYear,
-    pending: !!res.Pending,
-    age: res.AgeOrDateOfBirth.replace(/^0+/, '') || '0',
+    //id: res.CertificateID.toString(),
+    lastName: res['last_name'],
+    firstName: res['first_name'],
+    dogName: res['animal_name'],
+    id: res['animal_id'][0],
+    year: res['tag_date'],
+    personid: res['person_id'][0],
+    tagNo: res['tag_no'],
+    tagType: res['tag_type'],
+    tagDate: res['tag_date'],
+    tagExp: res['tag_exp'],
+    tagStat: res['tag_stat'],
+    vacDate: res['vac_date'],
+    vacTerm: res['vac_term'],
+    vacExp: res['vac_exp'],
+    userid: res['userid'][0],
```

Comment:
probably should be `userId` on the output side.

Also, I'd recommend grouping all of the `[0]` fields together and dropping in a comment about how they come back as arrays of duplicate values because they’re join columns in the `SELECT` statement.